lib/al/Library/Audio/AudioDirector.h

```diff
@@ -0,0 +1,80 @@
+#pragma once
+
+#include "al/Library/Area/AreaObjDirector.h"
+#include "al/Library/Bgm/BgmDirector.h"
+#include "al/Library/HostIO/HioNode.h"
+#include "al/Library/Se/SeDirector.h"
```

Comment:
forward-decl

Reply:
SeDirectorInitInfo needed by AudioDirectorInitInfo

Reply:
Alright, didn't see that - ignore.